src/models/user-played-time-response.model.ts

```diff
@@ -0,0 +1,44 @@
+import type { GameConcept } from "./title-concept.model";
+import type { Media } from "./game-media.model";
+
+export interface GetUserPlayedTimeResponse {
+  /** The title ID of this game 
+      @example CUSA26431_00
+      */
+  titleId: string;
+
+  /** Default name for the title */
+  name: string;
+
+  /** Localized name for the title */
+  localizedName: string;
+
+  /** URL to game's console launch screen, from image.api.playstation.com */
+  imageUrl: string;
+
+  /** URL to localized game's console launch screen. Will default to imageUrl if not unique */
+  localizedImageUrl: string;
+
+  /** Type of game being played. Can be ps5_native_game, ps4_game or none */
+  category: string;
```

Comment:
```ts
category: "ps5_native_game" | "ps4_game" | "none";
```

Is this an improvement?

Reply:
I never wrote typescript before this PR so I can't really say. I assume this is the strong typing you prefer to have?

Reply:
Yes, if the three possible values are known we should strongly-type them.